atlascope/core/models/dataset.py

```diff
@@ -15,7 +14,7 @@ class Dataset(TimeStampedModel, models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255)
     description = models.TextField(max_length=5000, blank=True)
-    content = S3FileField(null=True)
+    content = models.FileField(null=True, upload_to=settings.DATASET_STORAGE_DIR)
```

Comment:
```suggestion
    content = models.FileField(null=True)
```

This will default to [`MEDIA_ROOT`](https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-MEDIA_ROOT).